SparkPost/2web2ui
# SA-1956 - (UI) Don't allow "break down by" if comparing by that resource type

[SA-1956](https://sparkpost.atlassian.net/browse/SA-1956)

### What Changed

- `GroupByOption` filters by the current active comparison to remove it from the "Break Down By" select options when present

### How To Test

1. Go to `/signals/analytics` and add active comparisons
2. Click on the select labelled "Break Down By" and verify that the comparison type is not present in the list
3. Remove any active comparisons and verify that all select options render as expected

### To Do
